setup-machine.sh

```diff
@@ -137,6 +137,10 @@ sudo usermod -aG vx-group vx-services
 
 sudo usermod -aG video vx-ui
 
+# mark-scan requires access to the audio group
```

Comment:
@kofi-q does `mark` also require access? It doesn't seem like this config is specific to a single app so probably not an issue, but wanted to check.

Reply:
Sorry for the delay! Yeah, audio access is also needed for `mark`, so this would apply to that as well